### DISCUSSION TITLE
How to add and remove comment in xml quickly?

### DISCUSSION THREAD

**Discussion by iieeann** [WinMerge/winmerge#818] [General]
I love WinMerge file comparing function, but i can't figure out how to quickly comment and uncomment a xml line.  Typing one by one the comment syntax or delete them is very inconvenient.  

At the moment i have to open WinMerge alongside with Notepad++ to edit the file. Use WinMerge to see the difference, and use Notepadd++ to edit because the right click toggle comment function is not found in WinMerge. Is there an existing toggle comment function in winmerge or any plugin available for this purpose?

Thanks.

**Comment by sdottaka** [comment]
Unfortunately, WinMerge doesn't have a toggle comment function
